### ISSUE TITLE
2.0.x releases

### ISSUE THREAD

**Issue by jorisvandenbossche** [shapely/shapely#1742]
I released 2.0.1 today with a first batch of regression/bug fixes (https://github.com/shapely/shapely/releases/tag/2.0.1). 

There are still a few regressions/bugs to be tackled for 2.0.x (see current list in the 2.0.2 milestone: https://github.com/shapely/shapely/milestone/55), so I would propose to do a 2.0.2 shortly once a few of those are fixed.

**Comment by idanmiara** [comment]
Are we considering small enhancements in minor version updates? 
If so, suggesting to consider marking the following for `2.0.1` (or otherwise - for `2.1`): 
https://github.com/shapely/shapely/pull/1231
https://github.com/shapely/shapely/pull/1681

**Comment by jorisvandenbossche** [comment]
I am planning to do another 2.0.x release (2.0.3), mainly to ensure we have wheels that are "compatible" with the upcoming numpy 2.0 release (i.e. pin to <2 to avoid build compatibility issues), see https://github.com/shapely/shapely/issues/1972 for all details.

In addition, this also allows us to release wheels with the latest GEOS bug-fix release of the 3.11.x cycle (3.11.3; I think the idea is that GEOS 3.12 would be kept for shapely 2.1)

cc @shapely/shapely-team

**Comment by sgillies** [comment]
Great. Thanks @jorisvandenbossche !

**Comment by mwtoews** [comment]
Awesome, thanks! The one main thing that comes to mind is that GitHub recently introduced [native macOS Sillicon runners for open source projects](https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/). Some refactoring will need to enable it, and should resolve this warning:
> [Build arm64 wheels on macos-11](https://github.com/shapely/shapely/actions/runs/7874085014)
While arm64 wheels can be built on x86_64, they cannot be tested. The ability to test the arm64 wheels will be added in a future release of cibuildwheel, once Apple Silicon CI runners are widely available. To silence this warning, set `CIBW_TEST_SKIP: "*-macosx_arm64"`.

I'll see if I can tackle this one...

**Comment by jorisvandenbossche** [comment]
Yes, it would be nice to make use of that new feature. However, specifically for 2.0.x, I wouldn't consider that as critical, given that we already have wheels for MacOS arm64

**Comment by jorisvandenbossche** [comment]
@mwtoews I decided to not backport the macOS wheel PR. Since it's not strictly needed (we already have arm64 wheels), I prefer to keep packaging changes for the maintenance branch to a minimum. And normally removing the universal wheels should not matter, but in case someone does rely on its presence for some reason (maybe in some kind of repackaging workflow), then I think it's nicer to not drop them in a bug-fix release.

**Comment by jorisvandenbossche** [comment]
Released! https://github.com/shapely/shapely/releases/tag/2.0.3 / https://pypi.org/project/shapely/2.0.3/#files

**Comment by jorisvandenbossche** [comment]
I am planning to do a 2.0.4 release next week to have wheels compatible with numpy 2.0

**Comment by jorisvandenbossche** [comment]
A 2.0.6 was released yesterday to fix a compatibility issue with numpy 2.1.0, and at once we now also already provide Python 3.13 wheels.

**Comment by jorisvandenbossche** [comment]
Closing this issue now we are releasing 2.1.x, and we likely won't do more 2.0.x releases (unless something very specific comes up)
